components/mission/MissionInfo.vue

```diff
@@ -0,0 +1,65 @@
+<template>
```

Comment:
Rename the component to Info.
No need to declare it as MissionInfo since its already under mission component directory.